src/printer-postcss.js

```diff
@@ -339,13 +339,20 @@ function genericPrint(path, options, print) {
         return group(indent(fill(res)));
       }
 
+      const declaration = path.getParentNode(2);
+      const isMap =
+        declaration.type === "css-decl" && declaration.prop.startsWith("$");
+
```

Comment:
Might want to be a bit defensive and prefix with `declaration && `. It might not exist?

Reply:
Good call - just updated.